src/cli/bin.ts

```diff
@@ -172,195 +102,48 @@ yargs
       });
       args.implies("stack", "debug");
     },
-    async ({
+    ({
       inputPath,
       outputPath,
       serversPath,
       fromVersion,
       toVersion,
-      removeWidgets: keysOfWidgetPluginsToRemove,
+      removeWidgets,
       debug,
       stack,
-      onError: behaviorOnError,
+      onError,
     }) => {
-      const contentServer: ContentRecord = await fs.readJSON(inputPath);
-
-      const originalDashboardsContent = getContent(
-        contentServer,
-        "dashboard",
-        fromVersion,
-      );
-      const originalWidgetsContent = getContent(
-        contentServer,
-        "widget",
-        fromVersion,
-      );
-      const originalFiltersContent = getContent(
-        contentServer,
-        "filter",
-        fromVersion,
-      );
-
-      const servers: {
-        [serverKey: string]: { dataModel: DataModel<"raw">; url: string };
-      } = await fs.readJSON(serversPath);
-
-      const counters = _fromPairs(
-        [
-          "dashboards",
-          "widgets",
-          "filters",
-          "folders",
-          "calculated_measures",
-        ].map((type) => [
-          type,
-          {
-            success: 0,
-            partial: 0,
-            failed: 0,
-            removed: 0,
-          },
-        ]),
-        // _fromPairs returns a Dictionary.
-        // In this case, the keys used correspond to the attributes of OutcomeCounters.
-      ) as OutcomeCounters;
-      const errorReport = {};
-
       const doesReportIncludeStacks = stack;
 
-      // Handle the special case of 4.3 to 5.0 separately, as:
-      // - the corresponding migration function has a different signature than all others
-      // - in particular, it is the only migration step with async logic
-      if (fromVersion === "4.3") {
-        await migrate_43_to_50(contentServer, {
-          errorReport,
-          counters,
-          servers,
-          keysOfWidgetPluginsToRemove,
-          doesReportIncludeStacks,
+      const fileExtension = getFileExtension(inputPath);
+      const { fromVersion: validFromVersion, toVersion: validToVersion } =
```

Comment:
The `fromVersion` and `toVersion` command options has choices attached to them, making them already valid, thus no need to revalidate them.
Also see the `check` call at the end of the file.

Reply:
I changed the `validateVersions` to convert the potential given Atoti versions in the command line in order to only manipulate AUI versions in the codebase and enable our user to mix Atoti and AtotiUI versions.
Let me know if that works for you.